File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.java
// Single-line comment
/* Multi-line
   comment */
/** Javadoc comment */
package com.example;

@Annotation(name = "test")
public class Main<T> extends SuperClass implements Runnable {
    private static final int CONSTANT = 100;

    public static void main(String[] args) {
        int x = 10_000;
        List<String> list = new ArrayList<>();
        System.out.println("Hello");
    }

    @Override
    public <T> T genericMethod(T param) throws IOException {
        if (param instanceof String s) {
            return s;
        }
        return null;
    }
}

File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.rb
# Single-line comment
=begin
Multi-line comment
=end

module MyModule
    class Person
      include Enumerable
      attr_accessor :name

      @@class_var = 10

      def initialize(name)
        @name = name
        @counter = 0
      end

      def each(&block)
        [1,2,3].each(&block)
      end

      def self.create
        new("default")
      end
    end
  end

  lambda = ->(x) { x ** 2 }
  hash = { key: :value, 'old' => 1 }
  symbol = :test
  string = "Interpolation: #{name}"
  %w[array of words]
